src/components/FindMovie/FindMovie.tsx

```diff
@@ -17,20 +32,60 @@ export const FindMovie: React.FC = () => {
               id="movie-title"
               placeholder="Enter a title to search"
               className="input is-dander"
+              value={title}
+              onChange={event => {
+                setTitle(event.target.value);
+                setShowError(false);
+                setCanFind(true);
+              }}
             />
           </div>
 
-          <p className="help is-danger" data-cy="errorMessage">
-            Can&apos;t find a movie with such a title
-          </p>
+          {showError && (
+            <p className="help is-danger" data-cy="errorMessage">
+              Can&apos;t find a movie with such a title
+            </p>
+          )}
         </div>
 
         <div className="field is-grouped">
           <div className="control">
             <button
               data-cy="searchButton"
               type="submit"
-              className="button is-light"
+              className={
+                classNames('button', 'is-light', { 'is-loading': loading })
+              }
+              onClick={async event => {
+                event.preventDefault();
+                setCanFind(false);
+                setCurrentMovie(null);
+                setLoading(true);
+
+                try {
+                  const response = await getMovie(title);
+
+                  if ('Error' in response) {
+                    setShowError(true);
+
+                    return;
+                  }
+
+                  setCurrentMovie({
+                    title: response.Title,
+                    description: response.Plot,
+                    imgUrl: response.Poster === 'N/A'
+                      // eslint-disable-next-line max-len
+                      ? 'https://via.placeholder.com/360x270.png?text=no%20preview'
+                      : response.Poster,
+                    imdbUrl: `https://www.imdb.com/title/${response.imdbID}`,
+                    imdbId: response.imdbID,
+                  });
+                } finally {
+                  setLoading(false);
+                }
+              }}
+              disabled={!canFind || !title}
             >
               Find a movie
```

Comment:
need to change to ''Search again'', if movie wasn't add to list (https://prnt.sc/Axr2gPusoYOM)